http/http-minimum-reactive/src/test/java/io/quarkus/ts/http/minimum/reactive/HttpMinimumReactiveIT.java

```diff
@@ -15,4 +16,15 @@
     public void httpServer() {
         given().get("/api/hello").then().statusCode(HttpStatus.SC_OK).body("content", is("Hello, World!"));
     }
+
+    @Test
+    public void httpClusterConfig() {
```

Comment:
What is the added value of this test, compared to the `httpServer` test? If it is just the coverage of hierarchical data structure and Jackson annotations, then the data structure seems unnecessarily complicated to me.

Why is the test method called `httpClusterConfig`? I would name it accordingly to the point of the test (see previous question).

Do we want it also in the non-reactive module `http/http-minimum`?

Reply:
[The issue](https://github.com/quarkusio/quarkus/issues/20200) only talks about reactive, so doesn't apply to classic scenarios. 

As you are pointing the additional value is the data structure more in detail the list serialization os a "complex" pojo. I prefer to leave the data structure as it is because is a mirror of the origin reproducer.

Tag was added and test renamed to `nativeListSerialization`